### PR TITLE
refactor: Move all svgs to Icon component

### PR DESCRIPTION
**Pull request by thisisamir98**

### CODE REVIEW

File: src/script/components/calling/FullscreenVideoCall.tsx [hunk 1]
```diff
@@ -147,23 +147,17 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
...
-                  dangerouslySetInnerHTML={{__html: SVGProvider['message-unread-icon']?.documentElement?.innerHTML}}
                 />
               ) : (
-                <svg
-                  width={16}
-                  height={16}
-                  viewBox="0 0 16 16"
-                  dangerouslySetInnerHTML={{__html: SVGProvider['message-icon']?.documentElement?.innerHTML}}
-                />
+                <NamedIcon name="message-icon" width={16} height={16} />
```
Comment (Yserz) [inline src/script/components/calling/FullscreenVideoCall.tsx:160]: Width & height can be omitted here.

File: src/script/components/calling/FullscreenVideoCall.tsx [hunk 2]
```diff
@@ -175,12 +169,7 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
               css={isMuted ? videoControlActiveStyles : undefined}
               data-uie-name="do-call-controls-video-call-mute"
             >
-              <svg
-                width={16}
-                height={16}
-                viewBox="0 0 16 16"
-                dangerouslySetInnerHTML={{__html: SVGProvider['mic-off-icon']?.documentElement?.innerHTML}}
-              />
+              <NamedIcon name="mic-off-icon" width={16} height={16} />
```
Comment (Yserz) [inline src/script/components/calling/FullscreenVideoCall.tsx:172]: Width & height can be omitted here.

File: src/script/components/calling/FullscreenVideoCall.tsx [hunk 3]
```diff
@@ -227,12 +211,7 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
               data-uie-enabled={canShareScreen ? 'true' : 'false'}
               data-uie-name="do-toggle-screen"
             >
-              <svg
-                width={16}
-                height={16}
-                viewBox="0 0 16 16"
-                dangerouslySetInnerHTML={{__html: SVGProvider['screenshare-icon']?.documentElement?.innerHTML}}
-              />
+              <NamedIcon name="screenshare-icon" width={16} height={16} />
```
Comment (Yserz) [inline src/script/components/calling/FullscreenVideoCall.tsx:214]: Width & height can be omitted here.

File: src/script/components/participantAvatar/ServiceAvatar.tsx [hunk 4]
```diff
@@ -61,16 +60,16 @@ const ServiceAvatar: React.FunctionComponent<ServiceAvatarProps> = ({assetReposi
         }}
         data-uie-name="element-avatar-service-icon"
       >
-        <svg
+        <NamedIcon
+          name="service-icon"
           viewBox="0 0 32 32"
```
Comment (Yserz) [inline src/script/components/participantAvatar/ServiceAvatar.tsx:65]: Can this be width & height 32?

Comment (thisisamir98) [reply src/script/components/participantAvatar/ServiceAvatar.tsx:65]: nope, only viewBox is needed and not width attribute on element

**Pull request merged**